### ISSUE TITLE
Add support for exporting various types of private keys

### ISSUE THREAD

**Issue by hozzjss** [blockstack/blockstack-app#3530]
Folks using [ordswap](https://ordswap.io/import) are wondering about the possibility for the wallet to integrate the hex key feature like in ordswap

I haven't managed to find out how they do key generation or how to use the private keys they generate to get to a taproot address yet

Here's an example key: `59f060d4c4b67ce92a2030af8fd1017a813e349a4d1aba4ec4a6de940ae1d4ee`
The corresponding address generated by ordswap is: `bc1ptl6d7v2ye7uylxw9y70njkv55mg6r4yzx69lful70u9z8gg8n2hqegmd0y`

Please check this [support ticket](https://ptb.discord.com/channels/1075856098269089834/1090579498858197042) for context

**Comment by 314159265359879** [comment]
Example
https://medium.com/@hello_90591/importing-ordinals-wallet-into-ordswap-and-unisat-extension-e8c0330ef478

Additionally, would you want the wallet to display this in addition to the Mnemonic when revealing the Secret Key? if possible that is.
![image](https://user-images.githubusercontent.com/33360391/231597783-e3128fe3-9f26-491f-bee5-d277c0c5f5b7.png)

**Comment by 314159265359879** [comment]
Some users also show in interest in getting the private key of specific addresses in the Hiro wallet. For use of the address in other wallets.

**Comment by markmhendrickson** [comment]
There seem like two possible use cases raised in this issue:

1. User wants to import Hiro wallet into Ordswap (which only accepts hex keys atm, not seeds). We could support hex keys for this, though that team is interested in integrating Hiro properly as a connected wallet, so perhaps that's the more durable solution?
2. Users want to import specific addresses into other wallets... @314159265359879 not sure why they'd want this or how it'd work exactly? Is there a specific wallet users want to do this with?

**Comment by 314159265359879** [comment]
Perhaps this issue should be renamed to "add support for address private keys" in the Hiro wallet because that is what users want. Instead of exposing all their accounts with the master seed they prefer to load just a single account/address private key into another wallet instead. To limit risks. 

Today users are asking for this feature because they want to have more options with their Stamps so to access their native segwit bitcoin account in free wallet. What users really want though is stamp sending/listing etc in the Hiro wallet. 

The current workaround is to add the seed to Electrum and then get the secret key for an account from there.

**Comment by markmhendrickson** [comment]
Have any of you heard demand support for WIP as well? https://river.com/learn/terms/w/wallet-import-format-wif

**Comment by 314159265359879** [comment]
Yes, I have also seen requests to support WIF keys. In general users appreciate having the options to access the exact derivation path of an address and the HEX, WIF, private key, the more options we offer for them to secure their access through other means the more secure/certain they feel about the Hiro wallet.

Recently users needed their private key because a wallet that supports more features with stamps required it to access an address. It is a quite the process to get it via a third party wallet. Probably safer if the wallet supported this. Something for the advanced options menu as to not confuse users who do not need it?

**Comment by markmhendrickson** [comment]
With command line? https://github.com/hirosystems/wallet/issues/3400

- Wallet private key
- Address-specific private key

Hex (default 256-bit) or WIP formats
